### DISCUSSION TITLE
Changelog missing?

### DISCUSSION THREAD

**Discussion by osa1** [tree-sitter/tree-sitter#1031] [General]
Does this repo has a changelog? [Releases page](https://github.com/tree-sitter/tree-sitter/releases) does not show changes, and I couldn't find any files with name "changelog" or "CHANGELOG".

**Comment by osa1** [comment]
I saw the " Create a CHANGELOG file and start maintaining it" item in https://github.com/tree-sitter/tree-sitter/issues/930, so I guess it doesn't exist today but it's planned. Closing.

**Comment by maxbrunsfeld** [comment]
Yeah, I really apologize for going so long without setting this up. I do think it's important and hope to backfill a changelog pretty soon.

**Reply by osa1** [reply]
No need to apologize. tree-sitter is awesome, I enjoy using it both as a user of the existing parsers (writing queries etc.) and also as a parser developer, so thanks for doing this!

**Comment by madskjeldgaard** [comment]
Hi - sorry to bump this but a changelog would be realllllly appreciated. The latest version(s) have broken some things for me and it would be great to see what has changed when debugging

**Comment by ahlinc** [comment]
Starting from [v0.20.8](https://github.com/tree-sitter/tree-sitter/releases/tag/v0.20.8) I've started to maintain description for releases _(thanks for Github automation)_. The changelog file will be added in the future.
